### PR TITLE
📖 `Security-Policy`: Link to responsible disclosure guidelines for remediation

### PR DESCRIPTION
**Pull request by laurentsimon**
docs/checks.md: updated the doc
docs/checks/internal/checks.yaml: updated the source of truth for docs

Link to the responsible disclosure guidelines.

No breaking changes.

### CODE REVIEW

File: docs/checks.md [hunk 1]
```diff
@@ -517,7 +517,7 @@ information about a bug is not publicly visible.
 
 **Remediation steps**
 - Place a security policy file `SECURITY.md` in the root directory of your repository. This makes it easily discoverable by a vulnerability reporter.
-- The file should contain information on what constitutes a vulnerability and a way to report it securely (e.g. issue tracker with private issue support, encrypted email with a published public key).
+- The file should contain information on what constitutes a vulnerability and a way to report it securely (e.g. issue tracker with private issue support, encrypted email with a published public key). Follow the  [coordinated vulnerability disclosure guidelines](https://github.com/ossf/oss-vulnerability-guide/blob/main/guide.md) to respond to vulnerability disclosures.
```
Comment (justaugustus) [inline docs/checks.md:520]:
```suggestion
- The file should contain information on what constitutes a vulnerability and a way to report it securely e.g., an issue tracker with private issue support, encrypted email with a published public key. Follow the [coordinated vulnerability disclosure guidelines](https://github.com/ossf/oss-vulnerability-guide/blob/main/guide.md) to respond to vulnerability disclosures.
```

**Review by justaugustus — CHANGES_REQUESTED** [review]
@laurentsimon -- Thanks for adding this!

Microscopic nits:
- `e.g.,` needs a comma and shouldn't be parenthetical
- extraneous space

Added a code suggestion for both!

**Review by justaugustus — COMMENTED** [review]
@laurentsimon -- Another nit to briefly continue the convo from https://github.com/ossf/scorecard/pull/1532#issuecomment-1022481885:

The notes you left in the PR description are much clearer!
> docs/checks.md: updated the doc
docs/checks/internal/checks.yaml: updated the source of truth for docs

What I was suggesting in the previous PR was to make these not the PR description, but the actual commit messages.

That way, when the PR content gets squashed and merged, the details get included as part of the git history.

**Comment by laurentsimon** [comment]
> @laurentsimon -- Another nit to briefly continue the convo from [#1532 (comment)](https://github.com/ossf/scorecard/pull/1532#issuecomment-1022481885):
> 
> The notes you left in the PR description are much clearer!
> 
> > docs/checks.md: updated the doc
> > docs/checks/internal/checks.yaml: updated the source of truth for docs
> 
> What I was suggesting in the previous PR was to make these not the PR description, but the actual commit messages.
> 
> That way, when the PR content gets squashed and merged, the details get included as part of the git history.

Gocha. The info also seems useful for the PR reviewer. So need them in both places?

**Comment by laurentsimon** [comment]
> > @laurentsimon -- Another nit to briefly continue the convo from [#1532 (comment)](https://github.com/ossf/scorecard/pull/1532#issuecomment-1022481885):
> > The notes you left in the PR description are much clearer!
> > > docs/checks.md: updated the doc
> > > docs/checks/internal/checks.yaml: updated the source of truth for docs
> > 
> > 
> > What I was suggesting in the previous PR was to make these not the PR description, but the actual commit messages.
> > That way, when the PR content gets squashed and merged, the details get included as part of the git history.
> 
> Gocha. The info also seems useful for the PR reviewer. So need them in both places?

do you have a doc on updating a past commit? I can try adding it

File: docs/checks.md [hunk 2]
```diff
@@ -517,7 +517,7 @@ information about a bug is not publicly visible.
 
 **Remediation steps**
 - Place a security policy file `SECURITY.md` in the root directory of your repository. This makes it easily discoverable by a vulnerability reporter.
-- The file should contain information on what constitutes a vulnerability and a way to report it securely (e.g. issue tracker with private issue support, encrypted email with a published public key).
+- The file should contain information on what constitutes a vulnerability and a way to report it securely e.g., an issue tracker with private issue support, encrypted email with a published public key. Follow the [coordinated vulnerability disclosure guidelines](https://github.com/ossf/oss-vulnerability-guide/blob/main/guide.md) to respond to vulnerability disclosures.
```
Comment (olivekl) [inline docs/checks.md:520]: Tiny nit: if you're going to take the examples out of the parentheses, just be sure to put a comma before it.

**Review by olivekl — APPROVED** [review]
LGTM!

File: docs/checks.md (hunk 2 above)
Comment (justaugustus) [reply docs/checks.md]: > Tiny nit: if you're going to take the examples out of the parentheses, just be sure to put a comma before it.

hehe, yep: https://github.com/ossf/scorecard/pull/1545#discussion_r793877212

**Review by justaugustus — APPROVED** [review]
> do you have a doc on updating a past commit? I can try adding it

@laurentsimon -- Opened a general tracking issue for contribution doc improvements: https://github.com/ossf/scorecard/issues/1553

No need to worry about it here; I can tweak the merge message!

**Comment by justaugustus** [comment]
> I can tweak the merge message!

nvm, auto-merge was enabled 🙃

**Comment by laurentsimon** [comment]
> > I can tweak the merge message!
> 
> nvm, auto-merge was enabled 🙃

oops sorry.

**Pull request merged**